resources/js/Admin/Campaigns/Index.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from 'react';
+import classnames from 'classnames';
+
+import Layout from '../components/Layout';
+import { Inertia } from '@inertiajs/inertia';
+import Table from '../../components/Table/Table';
+import SearchField from '../components/Form/SearchField';
+import TabbedNav from '../components/TabbedNav/TabbedNav';
+import Pagination from '../../components/Pagination/Pagination';
+
+const Index = ({ campaigns }) => {
+  const urlParam = route().params.search;
+
+  const [query, setQuery] = useState(urlParam || '');
+
+  const [isLoading, setIsLoading] = useState(false);
+
+  const campaignList = campaigns.data;
+
+  const { meta } = campaigns;
+
+  const submitSearch = () => {
+    setIsLoading(true);
+
+    Inertia.get(route('admin.campaigns.index'), query ? { search: query } : {});
+  };
+
+  // Allow submitting search query immediately by pressing "enter" on keyboard.
+  const handleSubmit = event => {
+    event.preventDefault();
+
+    submitSearch();
+  };
+
+  // Links for internal tabbed page navigation.
+  const tabbedNavLinks = [
+    {
+      isDefault: true,
+      optionalParam: 'search',
+      title: 'Search',
+      routeName: 'admin.campaigns.index',
+    },
+    {
+      title: 'Open Campaigns',
+      routeName: 'admin.campaigns.index',
+      routeParams: { status: 'open' },
+    },
+    {
+      title: 'Closed Campaigns',
+      routeName: 'admin.campaigns.index',
+      routeParams: { status: 'closed' },
+    },
+  ];
+
+  return (
+    <div className="base-12-grid mx-auto py-8">
+      <h1 className="col-span-9 mb-10 text-purple-500 text-base uppercase">
+        Campaigns
+      </h1>
+
+      <a
+        className="bg-white hover:bg-blurple-500 border border-blurple-300 hover:border-blurple-500 col-span-3 font-bold mb-10 px-4 py-2 rounded text-blurple-300 hover:text-white text-center"
+        href="/admin/campaigns/create"
+      >
+        + New Campaign
+      </a>
+
+      <h2 className="col-span-full font-bold mb-10 text-black text-xl">
+        Campaigns List
+      </h2>
+
+      <TabbedNav links={tabbedNavLinks} />
+
+      {route().params.status ? null : (
+        <form
+          aria-label="Search for clubs by name"
```

Comment:
I think this needs to be updated to `campaigns` from `clubs`!
```suggestion
          aria-label="Search for campaigns by name"
```